### PR TITLE
Adds diesel config

### PR DESCRIPTION
**Pull request by barun511**
Diesel migrations work! Now diesel automatically creates databases based on our schema.rs file so we don't have to manually do funky stuff anymore

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ To stop world-wide inting.
 
 To develop, 
 
-1. Create a file called env.txt, copied from env.example.txt
+1. Create a file called .env, copied from .example.env.
```
Comment (barun511) [inline README.md:6]: For some reason, I can interpolate variables from .env but not from env.txt....

File: compose.yml [hunk 2]
```diff
@@ -1,13 +1,27 @@
...
   data_service:
+    depends_on:
+      database:
+        condition: service_healthy
     build: data_service
-    entrypoint: cargo watch -x run
+    command: sh -c "
+      diesel setup &&
+      diesel migration generate --diff-schema create_match_data &&
+      diesel migration run;
```
Comment (barun511) [inline compose.yml:10]: Semicolon here because we want to continue even if the migration fails

File: compose.yml [hunk 3]
```diff
@@ -1,13 +1,27 @@
 services:
   data_service:
+    depends_on:
+      database:
+        condition: service_healthy
     build: data_service
-    entrypoint: cargo watch -x run
+    command: sh -c "
+      diesel setup &&
+      diesel migration generate --diff-schema create_match_data &&
```
Comment (barun511) [inline compose.yml:9]: Moved migration steps here so we can diff against current schema and fix database if we need to

Comment (barun511) [reply compose.yml:9]: Quick flag that in times of trouble we can check our migration manually by commenting out the migration run here and then inspecting the migrations file manually to see if it makes sense.

File: README.md (hunk 1 above)
Comment (pyctal) [reply README.md:6]: find reason

**Review by pyctal — APPROVED** [review]
Not amazing but it'll pass for now.

**Pull request merged**